__tests__/integration.test.js

```diff
@@ -96,18 +96,56 @@ describe("GET: /api/articles", () => {
       .get("/api/articles")
       .expect(200)
       .then(({ body }) => {
-        const articles  = body.articles;
+        const articles = body.articles;
         articles.forEach((article) => {
-          expect(Object.keys(article).length).toBe(8)
+          expect(Object.keys(article).length).toBe(8);
           expect(typeof article.author).toBe("string");
           expect(typeof article.title).toBe("string");
-          expect(typeof article.article_id).toBe("number")
+          expect(typeof article.article_id).toBe("number");
           expect(typeof article.topic).toBe("string");
           expect(typeof article.created_at).toBe("string");
           expect(typeof article.votes).toBe("number");
-          expect(typeof article.article_img_url).toBe("string")
-          expect(typeof article.comment_count).toBe("string")
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number");
         });
       });
   });
 });
+
+describe("GET: /api/articles/:article_id/comments", () => {
+  test("GET: responds with a 200 status code, with an array of comments objects for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const commentsArticleId = body.comments;
```

Comment:
You should add a length check here to check you're not getting a false positive if you get a empty array of comments back